examples/common/scripts/vtadmin-down.sh

```diff
@@ -2,8 +2,17 @@
 
 source "$(dirname "${BASH_SOURCE[0]:-$0}")/../env.sh"
 
```

Comment:
We should add a brief comment just to clarify what parameters are expected with an example. For example, see here: https://github.com/vitessio/vitess/blob/main/examples/common/lib/utils.sh

Reply:
Yes but no. IMHO, it is clear with the variable assignment, and this is why I did not use `$1` and $`2` deep in the function, but feel free to do it yourself if you think it is worth your time.